ext/date/tests/bug72096.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Bug #72096 Swatch time value incorrect for dates before 1970
+--FILE--
+<?php
+for ($unix = 1461283200; $unix <= 1461369600; $unix += 8000) {
+	echo "Time:", gmdate('Y-m-d H:i:s = B',$unix), " Time:", gmdate('Y-m-d H:i:s = B',$unix-2592000000), PHP_EOL;
+}
+?>
+--EXPECT--
+Time:2016-04-22 00:00:00 = 041 Time:1934-03-04 00:00:00 = 041
```

Comment:
I'd change the test to do the following:

 - [ ] output the pre-1970 one on the next line
 - [ ] explicityly set the timezone in an ```--INI--``` setting to ```UTC```
 - [ ] Not use a constant like "25920000000" but instead do "365 * 24 * 3600" (etc) to show where it comes from
 - [ ] Add a ```:``` between the ```Bug #72096``` and ``` Swatch time``` in the title